packages/flutter/lib/src/services/restoration.dart

```diff
@@ -89,6 +89,32 @@ typedef _BucketVisitor = void Function(RestorationBucket bucket);
 /// is always ready to go on the platform thread when the operating system needs
 /// it.
 ///
+/// ## State Restoration on iOS
```

Comment:
@shihaohong Once https://github.com/flutter/engine/pull/23495 has rolled into the framework, can you use these steps to enable it for the gallery to test if the instructions make sense? Thanks.